fairlearn/metrics/_base_metrics.py

```diff
@@ -81,20 +83,25 @@ def _get_labels_for_confusion_matrix(labels, pos_label):
     return unique_labels
 
 
-def true_positive_rate(y_true, y_pred, sample_weight=None, pos_label=None) -> float:
+def true_positive_rate(
+    y_true: ArrayLike,
+    y_pred: ArrayLike,
+    sample_weight: Optional[ArrayLike] = None,
+    pos_label=None,
+) -> float:
     r"""Calculate the true positive rate (also called sensitivity, recall, or hit rate).
 
     Read more in the :ref:`User Guide <custom_fairness_metrics>`.
 
     Parameters
     ----------
-    y_true : array-like
```

Comment:
array-like is pretty well defined. "ArrayLike" looks like Java 😁

Reply:
I could change to `array_like` - it would have to be a valid Python name

Reply:
why does it mean to be a valid python name? I've never had an issue in scikit-learn with this.

Reply:
I wonder, too... maybe explicitly ignored? Types are cased that way, though, aren't they?

Reply:
If you want to discuss this some more @adrinjalali  but are otherwise OK with this PR, I can remove this change, and we can discuss it separately. This was mainly supposed to be about getting CircleCI happy again.

Reply:
Ping @adrinjalali ?

Reply:
Yes, happy to remove this and merge with the rest of the PR.

Reply:
Removed @adrinjalali